<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кофейная карта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchInput">
        <property name="placeholderText">
         <string>Поиск...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="roastFilter">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Светлая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Средняя</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Темная</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeFilter">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>В зернах</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Молотый</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="coffeeTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Обжарка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Тип</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вкус</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Объем</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>